cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -139,6 +139,8 @@ public void testSessionsShareUserTask() {
 
     // The 2nd request should reuse the UserTask created for the 1st request since they use the same session and send the same request.
     Assert.assertEquals(1, userTaskManager.numActiveSessionKeys());
+    EasyMock.verify(mockUuidGenerator, mockHttpServletResponse1, mockHttpServletResponse2, mockHttpServletResponse3);
+    EasyMock.verify(mockHttpServletRequest1, mockHttpServletRequest2, mockHttpServletRequest3);
```

Comment:
Nit: Can we avoid splitting this into separate verify calls and use a single verify?

Reply:
If I make it simple, this Linter Error has occurred.

[ant:checkstyle] [ERROR] /home/circleci/workspace/cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java:142: Line is longer than 150 characters (found 192). Try to keep lines under 120 characters. [LineLength]

Reply:
We can put a line break within a single verify call -- something like:
```
EasyMock.verify(mockUuidGenerator, mockHttpServletResponse1, mockHttpServletResponse2, mockHttpServletResponse3, 
                mockHttpServletRequest1, mockHttpServletRequest2, mockHttpServletRequest3);
```